core/unit_test/tools/printing-tool.cpp

```diff
@@ -25,18 +25,20 @@ extern "C" void kokkosp_begin_parallel_for(const char* name,
                                            const uint32_t devID,
                                            uint64_t* kID) {
   *kID = parallel_for_id;
-  printf("kokkosp_begin_parallel_for:%s:%u:%lu::", name, devID, *kID);
+  printf("kokkosp_begin_parallel_for:%s:%u:%lu::", name, devID,
```

Comment:
If the goal is to get rid of warnings, should we use PRIu32, etc codes here instead?